libindy-crypto/src/cl/issuer.rs

```diff
@@ -340,7 +340,7 @@ impl Issuer {
         Ok((issuer_rev_pub_key, issuer_rev_priv_key))
     }
 
-    fn _calc_m2(prover_id: &str, rev_idx: Option<u32>) -> Result<BigNumber, IndyCryptoError> {
+    fn _gen_claim_context(prover_id: &str, rev_idx: Option<u32>) -> Result<BigNumber, IndyCryptoError> {
```

Comment:
Please put a comment that `claim_context` is denoted by `m2` in the paper

Reply:
Done